pages/common/makebuildserver.md

```diff
@@ -0,0 +1,12 @@
+# makebuildserver
+
+> Create an F-Droid build server virtual machine.
+> More information: <https://f-droid.org/en/docs/Build_Server_Setup/>.
+
+- Create a new virtual machine or update an existing one (if available):
+
+`makebuildserver`
+
+- Don't try to update an existing virtual machine and always create a new one:
```

Comment:
How about we skip the part about not trying to update an existing machine and just say: "Always create a fresh virtual machine" ?